### ISSUE TITLE
"Generate Method" for an event handler doesn't add nullability annotation

### ISSUE THREAD

**Issue by Tragetaschen** [dotnet/roslyn#39693]
**Version Used**:
16.3.7 / 16.4.0 Preview 3.0

**Steps to Reproduce**:
```csharp
#nullable enable
using System;
public class Test
{
    private event EventHandler<EventArgs>? myEvent;

    public void M() => myEvent += handler; // Run "Generate Method" here
}
```

Run the "Generate Method" quick fix for `handler` and note the nullability annotation on the `sender` parameter.

**Expected Behavior**:
```diff
+    private void handler(object? sender, EventArgs e) => throw new NotImplementedException();
```
To match the `EventHandler` delegate, `sender` should be typed `object?`.

**Actual Behavior**:
```diff
+    private void handler(object sender, EventArgs e) => throw new NotImplementedException();
```
The `sender` parameter is missing the nullable annotation resulting in an immediate warning.

![image](https://user-images.githubusercontent.com/1947968/68220841-becb4980-ffe8-11e9-831a-35371a5dccf5.png)

**Comment by jasonmalinowski** [comment]
It's possible this is fixed with #39498.

**Comment by jasonmalinowski** [comment]
Indeed, this is fixed with either #39498 or #39737. I'm testing with #39737 bits so not really sure, and not really worth figuring out which it is.
